specifyweb/specify/views.py

```diff
@@ -682,15 +682,26 @@ def progress(cur: int, additional_total: int=0) -> None:
 
     # Create a message record to indicate the finishing status of the record merge
     logger.info('Creating finishing message')
-    Message.objects.create(user=specify_user, content=json.dumps({
-        'type': 'record-merge-succeeded' if response.status_code == 204 else 'record-merge-failed',
-        'response': response.content.decode(),
-        'task_id': self.request.id,
-        'table': model_name.title(),
-        'new_record_id': new_model_id,
-        'old_record_ids': json.dumps(old_model_ids),
-        'new_record_data': json.dumps(new_record_info['new_record_data'])
-    }))
+    try:
+        Message.objects.create(user=specify_user, content=json.dumps({
+            'type': 'record-merge-succeeded' if response.status_code == 204 else 'record-merge-failed',
+            'response': response.content.decode(),
+            'task_id': self.request.id,
+            'table': model_name.title(),
+            'new_record_id': new_model_id,
+            'old_record_ids': json.dumps(old_model_ids),
+            'new_record_data': json.dumps(new_record_info['new_record_data'])
+        }))
+    except Exception as e:
+        # If an error occurs, create a message without the old_record_ids and new_record_data fields
+        logger.info(f"Error while creating message: {e}")
+        Message.objects.create(user=specify_user, content=json.dumps({
+            'type': 'record-merge-succeeded' if response.status_code == 204 else 'record-merge-failed',
+            'response': response.content.decode(),
+            'task_id': self.request.id,
+            'table': model_name.title(),
+            'new_record_id': new_model_id
+        }))
```

Comment:
Can we try casting the `old_record_ids` and `new_record_data` to strings (either with `str(obj)` or `repr(obj)`)? 
All Django objects have those properties defined, and some information is better than none.  
I'm unsure about SQLAlchemy, but I would be very surprised if they didn't have __repr___ defined _at the very least_ for their classes... 

Regardless, the frontend doesn't require `old_record_ids` or `new_record_data` for rending the notification, so outright removing the fields is probably the safe bet...

(Related: also see my comment https://github.com/specify/specify7/commit/0f8be18e197c72d7eba2242aeaea36b265c2be5b#r123323331)

Also, what errors (serialization or otherwise) was occurring around old_record_ids? Should just be an array of numbers, or an empty array.